Implement prediction reliability metric

WuestFlorian opened the pull request:
Closes https://github.com/SWT-Forschungsprojekt/planning/issues/26
Closes https://github.com/SWT-Forschungsprojekt/realtime-metrics/issues/11

WuestFlorian committed:
Implement prediction reliability metric

First implementation if the prediction reliability metric

WuestFlorian commented:
In the metric it is defined that only stop time updates created a maximum of 60 minutes before the planned first stop of the trip should be considered. This is not yet implemented.

1Maxnet1 commented on src/realtime_metrics/main.py:
```diff
@@ -395,6 +399,55 @@ def get_next_actual_arrival(timestamp: int, route_id: str, stop_id: str) -> Stop
+        lower_bound = -60 * numpy.log(time_to_prediction + 1.3)
+        upper_bound = 60 * numpy.log(time_to_prediction + 1.5)
+
+        if prediction_error > lower_bound and prediction_error < upper_bound:
```
Would simplify that:
```suggestion
        if lower_bound < prediction_error < upper_bound:
```

1Maxnet1 reviewed (approved):
Looks good to me, one minor improvement

maexled committed:
WIP: check with gtfsdb schedule

WuestFlorian committed:
Fixed actual arrival time updating

Fixed actual arrival time updating not working, round metrics, add logs and change log level to mostly debug

maexled committed:
get first trip arrival time and compare to trip timestamp

maexled committed:
Merge remote-tracking branch 'origin/feature/prediction-reliability' into feature/prediction-reliability

maexled committed:
remove unneeded code
